jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/OdeProperties.java

```diff
@@ -29,6 +29,7 @@
     /////////////////////////////////////////////////////////////////////////////
     // Kafka Topics
     public static final String KAFKA_TOPIC_J2735_BSM = "topic.J2735Bsm";
+    public static final String KAFKA_TOPIC_J2735_BSM_JSON = "topic.J2735BsmRawJSON";

```

Comment:
These being external interfaces, we should make them configurable just like the other properties but set to the above values by default .